recipes/bioconductor-scan.upc/LICENSE

```diff
@@ -0,0 +1,8 @@
+MIT License
+Copyright (c) 2012, Stephen R. Piccolo, Andrea H. Bild, and W. Evan Johnson
+
+Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the "Software"), to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions:
```

Comment:
Yes, this is just a recipe. If you want to ship the license with the package you should use: https://conda.io/docs/building/meta-yaml.html#license-file